tutorials/grid_coordinates.py

```diff
@@ -0,0 +1,289 @@
+"""
+.. _grid_coordinates:
+
+Grid coordinates
+====================
+
+Grid coordinates in Verde are used to create points on a regularly spaced grid
+that are then used in the spline method to interpolate between sample data
+points. The grid can be specified either by the number of points in each
+dimension (the *shape*) or by the grid node spacing.
+
+Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`
+function. It creates a set of regularly spaced points in both the west-east and
+south-north directions, i.e. a two-dimensional spatial grid.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import verde as vd
+from matplotlib.patches import Rectangle
+
+########################################################################################
+# First let's create a region that is 1000 units west-east and 1000 units south-north,
+# and we will set an initial spacing to 100 units.
+
+spacing = 100
+west, east, south, north = 0, 1000, 0, 1000
+region = (west, east, south, north)
+
+# create the grid coordinates
+easting, northing = vd.grid_coordinates(region=region, spacing=spacing)
+
+########################################################################################
+# We can check the dimensions of the grid coordinates to confirm that it is 11x11
+
+print(easting.shape, northing.shape)
+
+########################################################################################
+# Let's define two functions to visualize the region bounds and grid points
+
+
+def plot_region(ax, region=None):
+    "Plot the region as a solid line."
+    if region is not None:
```

Comment:
Since this function only plots the region, there is no need for the `region=None` default.

Reply:
removed the default